python/lsst/validate/base/__init__.py

```diff
@@ -5,15 +5,15 @@
 
 try:
     # version.py is auto-generated by sconsUtils
-    from .version import *  # noqa: F403
+    from .version import *  # noqa: F401, F403
```

Comment:
Do we really want all the noqa's in our `__init__` packages? Seems like we should just have them ignored entirely by linters, since they're special.